serializers/web/invoice.rb

```diff
@@ -6,6 +6,7 @@ def self.base(inv)
       ubid: inv.ubid,
       path: inv.path,
       name: inv.name,
+      filename: "Ubicloud-#{inv.begin_time.strftime("%Y-%m")}-#{inv.invoice_number}",
```

Comment:
why is this `begin_time`? can we use `created_at` instead? When I look at my invoices in my e-mails, they are generally named with the day of the invoice creation instead of the start date of the invoice.

Reply:
We generate invoices on the first day of the following month. For instance, the invoice for services used in September 1-30 is created on October 1st. This invoice reflects the usage for September.
We use the 'begin_time' into both the invoice name and number. Consequently, I have also use 'begin_time' in the filename.

Reply:
If we decide to use ﻿`created_at` as the invoice month, we will also need to adjust the name and number I think

Reply:
@byucesoy Given your deeper understanding of invoice properties, what is your opinion on this matter?

Reply:
I don't know if this has any accounting consequences. What about using the `end_date`, then? The reason why I'm asking is that it feels a little bit weird to have an invoice with the spendings in it is in a future date. This is simply a cosmetic thing, though.

Reply:
Since we are only using the year and month, the 'begin_time' and 'end_time' return the same value.

Reply:
I looked at Hetzner's invoices as an example. Hetzner uses the day they issue the invoice in their filename. That said, Hetzner charges upfront and we charge at the end of the month based on usage.

Reply:
I checked Crunchy and DigitalOcean, both of which operate similarly to us. I chose to proceed with this approach